tests/TestCase/View/Form/FormContextTest.php

```diff
@@ -53,7 +53,7 @@ public function setUp(): void
     public function testGetRequiredMessage()
     {
         $validator = new Validator();
-        $validator->requirePresence('title', true, 'Don\'t forget a title!');
+        $validator->notEmpty('title', 'Don\'t forget a title!');
```

Comment:
Seems like this entire test could be removed if it is for testing requirePresence() interactions with message generation.

Reply:
The test is required. Istead of requirePresence() message it does need to check for notEmpty() message. There are no other tests for the same.